src/Symfony/Component/HttpFoundation/Tests/ResponseHeaderBagTest.php

```diff
@@ -118,7 +118,7 @@ public function testToStringIncludesCookieHeaders()
 
         $bag->clearCookie('foo');
 
-        $this->assertContains('Set-Cookie: foo=deleted; expires='.gmdate('D, d-M-Y H:i:s T', time() - 31536001).'; path=/; httponly', explode("\r\n", $bag->__toString()));
+        $this->assertRegExp('#^Set-Cookie: foo=deleted; expires='.gmdate('D, d-M-Y H:i:s T', time() - 31536001).'; path=/; httponly#m', $bag->__toString());
```

Comment:
Btw, this change is not required for HHVM, but still, this test is transient (especially on HHVM?) and assertRegExp shows a nicer error message than assertContains in case of failure. That's the only reason for it.